[15.0][FIX] web_responsive: only change width of main control panel

vincent-hatakeyama opened the pull request:
The Search More panel is incorrectly displayed due to the reduced screen so only change width on the main screen.

Fixes #2395

pedrobaeza commented:
I don't think removing it is the right solution, but restricting it to the main screen, not the popup ones, which is the problem. That media query is done for very big displays (but as said, the main view).

SplashS commented on web_responsive/static/src/components/control_panel/control_panel.scss:
```diff
@@ -18,16 +18,6 @@
     }
     // For FULL HD devices
-    @media (min-width: 1900px) {
-        .o_cp_top_left,
```
As @pedrobaeza wrote it's better to avoid using this rule in popup rather than removing it at all.
You can change this part of code like this:
```scss
            .o_action_manager & .o_cp_top_left,
            .o_action_manager & .o_cp_bottom_left {
                width: 40%;
            }
            .o_action_manager & .o_cp_top_right,
            .o_action_manager & .o_cp_bottom_right {
                width: 60%;
            }
```

vincent-hatakeyama replied:
I’ve tested the solution, it looks like it works. I’ve kept the original width (the comment below was inverting them, I do not think that was intentional).

vincent-hatakeyama committed:
[FIX] web_responsive: only change width of main control panel

The Search More panel is incorrectly displayed due to the reduced screen
so only change width on the main screen.

Fixes #2395

SplashS reviewed (approved):
Tested on runboat.
Works as expected. Only main screen has shrunk width of search bar.

pedrobaeza commented:
/ocabot merge patch
